### PR TITLE
Added new transform to UCR

### PR DESCRIPTION
**Pull request by kkrampa**
@czue

### CODE REVIEW

File: corehq/apps/userreports/transforms/custom/date.py [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
 
 
 def get_month_display(month_index):
     return calendar.month_name[int(month_index)]
+
+
+def days_elapsed_from_date(date):
+    date = force_to_datetime(date)
+    now = datetime.utcnow()
+    return (now - date).days
```
Comment (czue) [inline corehq/apps/userreports/transforms/custom/date.py]: I think there are some problems with this approach. In particular, it assumes that all UCRs will be processed in ~real time, but if you are reindexing a new report then the current delta will be way off from the delta when the form was submitted.

I think what you need to do is date math - and the anchor date should be `received_on`, no? 

I don't think we can do any indicator that depends on the current time in a meaningful way.

Comment (kkrampa) [inline corehq/apps/userreports/transforms/custom/date.py]: @czue I think that this problem only occurs in date sources but I want to use this in report. Frankly, I thought that transform is only defined for reports. I agree that it doesn't make sense for data sources. 

Maybe I wasn't precise what I want to achieve. Basically I want to display in report how many days elapsed from some date stored in defined UCR data source.

Comment (czue) [inline corehq/apps/userreports/transforms/custom/date.py]: Oh I see - sorry we have two types of transforms and I was confused. This makes sense to me.

**Comment by czue** [comment]
can you add a few tests? then I think this is good to go.

**Pull request merged**